fe/fe-core/src/main/java/org/apache/doris/statistics/Statistics.java

```diff
@@ -131,6 +131,11 @@ public Statistics addColumnStats(Expression expression, ColumnStatistic columnSt
         return this;
     }
 
+    public boolean containUnknownChildStats() {
+        return expressionToColumnStats.values().stream()
+                .allMatch(e -> e.isUnKnown);
```

Comment:
we should check the columnStatistics of the Not.child(), not all the columns

Reply:
done